mips/index.md

```diff
@@ -6,46 +6,56 @@ group: "navigation"
 
 # Model Intercomparison
 
-#### [AMIP - Atmospheric Model Intercomparison Project][AMIP]
+### [AMIP - Atmospheric Model Intercomparison Project][AMIP]
 AMIP is a standard experimental protocol for global atmospheric general circulation models (AGCMs).
 
-#### [CMIP - Coupled Model Intercomparison Project][CMIP]
+### [CMIP - Coupled Model Intercomparison Project][CMIP]
 The Coupled Model Intercomparison Project (CMIP) studies output from coupled ocean-atmosphere general circulation models that also include interactive sea ice.
 
-#### [CMIP2][CMIP2]
+### [CMIP2][CMIP2]
 The Coupled Model Intercomparison Project Phase II
 
-#### [CMIP3][CMIP3]
+### [CMIP3][CMIP3]
 The Coupled Model Intercomparison Project Phase III
 
-#### [CMIP5][CMIP5]
+### [CMIP5][CMIP5]
 The Coupled Model Intercomparison Project Phase V
 
-#### [CMIP6][CMIP6]
+### [CMIP6][CMIP6]
 The Coupled Model Intercomparison Project Phase VI
 
+### [CMIP7][CMIP7]
+The Coupled Model Intercomparison Project Phase VII
+
 [Related Intercomparison Projects][related]
 
 ---
 
 # obs4MIPs
 
-#### [obs4MIPs][obs4MIPs]
+### [obs4MIPs][obs4MIPs]
 obs4MIPs (Observations for Model Intercomparisons Project) is an activity to make observational products more accessible for climate model intercomparisons
 
 ---
 
 # input4MIPs
 
-#### [input4MIPs][input4MIPs]
-input4MIPS (input datasets for Model Intercomparison Projects) is an activity to make available via ESGF the boundary condition and forcing datasets needed for CMIP6. Various datasets are needed for the pre-industrial control (piControl), AMIP, and historical,  simulations, and additional datasets are needed for many of the <a href="http://www.wcrp-climate.org/modelling-wgcm-mip-catalogue/modelling-wgcm-cmip6-endorsed-mips" target="_blank">CMIP6-endorsed model intercomparison projects (MIPs) experiments</a>. Earlier versions of many of these datasets were used in the <a href="https://pcmdi.llnl.gov/mips/cmip5" target="_blank">5th Coupled Model Intercomparison Project (CMIP5)</a>
+### [input4MIPs][input4MIPs]
+input4MIPs (input datasets for Model Intercomparison Projects) is an activity to make available via ESGF the boundary condition and forcing datasets needed for CMIP6 and CMIP7. The latest information about CMIP7 datasets can be found on the <a href="https://input4mips-cvs.readthedocs.io" target="_blank">input4MIPs_CVs website</a>.
+
+Various datasets are needed for the pre-industrial control (piControl), AMIP, and historical simulations, and additional datasets were needed for many of the <a href="https://wcrp-cmip.org/mips/cmip6-endorsed-mips" target="_blank">[CMIP6-endorsed model intercomparison projects (MIPs) experiments</a>, along with those that will be needed for the CMIP7 MIP follow-ons.
```

Comment:
```suggestion
Various datasets are needed for the pre-industrial control (piControl), AMIP, and historical simulations, and additional datasets were needed for many of the <a href="https://wcrp-cmip.org/mips/cmip6-endorsed-mips" target="_blank">CMIP6-endorsed model intercomparison projects (MIPs) experiments</a>, along with those that will be needed for the CMIP7 MIP follow-ons.
```
Remove stray bracket

Reply:
Great, that should be fixed in the updated commit